CMSgov/mint-app
# [MINT-2128] Update status modal

# MINT-2128

## Description

[Figma](https://www.figma.com/design/qXhO12xSSkDF4LaToGKJoV/MINT-Blue-Sky-Ideas?node-id=690-24765&t=LZkX0Ka81xu68h2E-0)

- Added a modal to the task list (Collab area when built) if the presence of an `suggestedPhase`
- Modal will also only render if the user has not viewed this modal in the current session
- Modal will render statuses choose from based on the phase - should match [Figma](https://www.figma.com/design/qXhO12xSSkDF4LaToGKJoV/MINT-Blue-Sky-Ideas?node-id=691-25495&t=59uOkSge6oRs6Y8S-0)
- Unit tests for session and modal


## How to test this change

1.  Seed db
2. Click on `Enhancing Oncology Model`, verify that the modal renders and does not render again once the modal is closed.  The modal should render again on re-login if the status was not updated
3. Test the other phases based on the criteria set in Figma

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
